src/components/accordion/accordion.js

```diff
@@ -95,7 +95,7 @@ class Accordion extends mixin(createComponent, initComponentBySearch, handles) {
    * The map associating DOM element and accordion UI instance.
    * @type {WeakMap}
    */
-  static components = new WeakMap();
+  static components /* #__PURE__ */ = new WeakMap();
```

Comment:
Are we safe doing this versus having a compiler/build tool like webpack insert these?

Reply:
@joshblack Good question, I made a change to rename the hint to avoid the conflict with `/* #__PURE__ */`.